ERROR at //build/config/compiler/BUILD.gn:2817:9 (//build/toolchain/linux/unbundle:default_for_rust_host_build_tools): Assignment had no effect.
        [ "{{target_out_dir}}/{{label_name}}/{{source_name_part}}.dwo" ]
        ^--------------------------------------------------------------
You set the variable "c_additional_outputs" here and it was unused before it went
out of scope.
See //build/config/BUILDCONFIG.gn:357:3: which caused the file to be 

Partially reverts commit deffd02cfa87a0f69cab81068e289d3df39fe875
Author: Takuto Ikuta <tikuta@chromium.org>
Date:   Sun May 10 19:55:19 2026 -0700

    Reland^3 "build: add .dwo outputs to cc and cxx tools for debug fission"


(which requires a newer generate-ninja than we have in trixie.)


--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -2811,10 +2811,6 @@ config("split_dwarf") {
   # thinlto requires -gsplit-dwarf in ldflags.
   if (use_thin_lto && !is_apple) {
     ldflags = split_dwarf_flags
-  } else {
-    # .dwo files are generated when ThinLTO is not used.
-    c_additional_outputs =
-        [ "{{target_out_dir}}/{{label_name}}/{{source_name_part}}.dwo" ]
   }
 }
 
